### DISCUSSION TITLE
[FEAT] Get IP of wifi module

### DISCUSSION THREAD

**Discussion by ad-on-is** [fastfetch-cli/fastfetch#1253] [Q&A]
Be sure to read [FAQ](https://github.com/fastfetch-cli/fastfetch?tab=readme-ov-file#faq) before submitting a new issue.

## Wanted features:
Is there a way to get the IP in the wifi-module... I tried all positions {1} to {11}, but none of them show the IP.

## Motivation:
I have a NET section, where I display my public IP as well as the IPs assigned to the network interfaces (ethernet and wifi)
![image](https://github.com/user-attachments/assets/1ba6431d-fd67-48c7-8ce7-3ec7943beba3)

**Comment by CarterLi** [comment]
`Wifi` module doesn't report its IP. `Local IP` module does.

```json
{
  "type": "localip",
  "defaultRouteOnly": false,
  "namePrefix": "interfaceNameOfTheWifi"
}
```
